.changeset/good-mirrors-attack.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Updated] Added updates to Engineers - Contributing page
```

Comment:
Can we be more specific? I think ambiguous terms like `added updates` makes it difficult to know what changes were made when we look back